### DISCUSSION TITLE
Custom notes

### DISCUSSION THREAD

**Discussion by FluxiaStargazer** [ppy/osu#16999] [Ideas] [answered]
Custom notes could make osu! maps a bit more interesting like...

- Notes that go at double the speed that you must hit or you die.
- Notes that you shouldn't hit.
- Notes that speeds up the notes' speed
...and more!

To show that it's different from normal notes, maybe we can make the notes differently shaped or colored or have patterns and stuff.

**Comment by bdach** [comment, marked as answer]
This discussion is too vague to be anything useful and pieces of this have been suggested before by multiple people. Pick and choose from the links below:

https://github.com/ppy/osu/discussions/13122
https://github.com/ppy/osu/discussions/13157
https://github.com/ppy/osu/discussions/16126
https://github.com/ppy/osu/discussions/13073
https://github.com/ppy/osu/discussions/13009

and probably several more that I'm missing.

Please use the search function in the future.
